### DISCUSSION TITLE
Score for Detection when assume_straight_pages is False

### DISCUSSION THREAD

**Discussion by WalidHadri-Iron** [mindee/doctr#1612] [General]
Hi,

When using the Detection part for inference with `assume_straight_pages=False` the output is list of `(x1, y1), (x2, y2), (x3, y3), (x4, y4)` but the score is missing, which is present when `assume_straight_pages=True` . This is coming from [here ](https://github.com/mindee/doctr/blob/9c92f5cbd0bf64706672c29bf2d43117815c1794/doctr/models/detection/differentiable_binarization/base.py#L153) I guess, by not appending the score when`assume_straight_pages=False,` . Same thing with Fast and Linknet.

I wonder if this could be a good addition?

Thanks,

**Comment by felixdittrich92** [comment]
Hi again @WalidHadri-Iron :wave:,

You are right currently we need the box conf score only in the detection post processing part.
Do you think it makes sense to add it to the Document output / or maybe only to the `detection_predictor` output ? 

We need to be a bit critical which information we add to the results to avoid littering things

**Reply by WalidHadri-Iron** [reply]
Hi @felixdittrich92,

Thanks for your quick answers.

I guess that's my poitn is that the output of the detection (after post-processing) should be equivalently the same whether the `assume_straight_pages` if False or True, why would we have the score for True and not False. Moreover it could be useful in some use cases where we study the detection separately to have the score. Thanks a lot !
